draft-ietf-quic-http.md

```diff
@@ -373,7 +374,7 @@ MUST be treated as a stream error of type HTTP_WRONG_STREAM_DIRECTION.
  0                   1                   2                   3
  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|Stream Type (8)|                  Push ID (i)                ...
+|    0x01 (8)   |                  Push ID (i)                ...
```

Comment:
The `(8)` suggests that this is an octet, whereas the stream type is a varint.  I'd make this figure use the same format as _fig-stream-header_.

Reply:
A varint whose value is specified, and therefore fits in one octet.  But point taken -- you're not required to use the minimal encoding, so this is overly restrictive.

Reply:
I reverted this particular commit and re-did it to use the explicit value but still show it as a varint, which seems the most accurate.  (I know @LPardue would prefer I remove it entirely.)